templates/index.html

```diff
@@ -45,16 +48,7 @@ <h3 class="masthead-brand">${title}</h3>
                 % endfor
 
                 <li class="dropdown">
```

Comment:
This whole dropdown entry can probably be removed from the top navbar now that the uls language selector is in the footer, no?